### ISSUE TITLE
Feature request: machine parseable output

### ISSUE THREAD

**Issue by slashbeast** [qarmin/czkawka#1006]
It would be fantastic if czkawka could output a machine parseable payload like json.

Running something like `czkawka_cli video -d /path/to/directory` results in an output that is hard to parse, I'd love to consume it with Python and for example remove the duplicates, for example, starting with the smallest, or run additional postprocesing to delete those that hve lowest resolution and leave the highest, even if czkawka wouldn't provide me such metadata, I could probe for them on each set.

While it is possible already to parse the output, it require some complex tier 3 hackery to get it done, I think it would make the most sense to have optional JSON output format

Thanks!

**Comment by lihuilai** [comment]
I have the same needs，optional JSON output format.
fclones provide that feature, so I have to do that for now.

**Comment by qarmin** [comment]
After recent changes(#1082) adding support for printing json output to file, should not be so hard(at least for most modes), so probably I will implement this soon(but not sure if this will be available in next version)
